src/components/SRNEntry.vue

```diff
@@ -91,6 +91,7 @@ import { redirectToDestination } from "@/services/redirectToDestination.js";
 import { sendSQSMessage } from "@/services/API/sqs";
 
 const numberOfSRNsAllowed = 10;
+const authType = "SRN";
```

Comment:
Actually, I let this be here because it'll soon be turned into a prop. I thought of doing it for this PR, but that would also change the URL format. So will do it along with the OTP auth component.